package.json

```diff
@@ -46,18 +46,16 @@
     "eslint-plugin-jest": "^25.0.0",
     "eslint-plugin-prettier": "^4.0.0",
     "husky": "^7.0.1",
-    "jest": "^27.2.4",
+    "jest": "^27.2.5",
     "jest-watch-typeahead": "^1.0.0",
     "lint-staged": "^12.0.0",
     "prettier": "^2.3.2",
     "pretty-format": "^27.2.4",
     "typescript": "^4.4.3"
   },
   "dependencies": {
-    "expect": "^26.6.2",
     "jest-diff": "^27.2.5",
-    "jest-get-type": "^27.0.6",
-    "jest-matcher-utils": "^27.2.4"
+    "jest-get-type": "^27.0.6"
   },
```

Comment:
Do we want to add `jest` as a `peerDependency` too? cc/ @SimenB do you know which patch the fix went in for asymmetric matchers?

Reply:
I looked for the tags with the commit with the fix, and it was version 27.2.5.

Reply:
We'd have to take it out of dev dependencies to do that, right?
```
warning "jest" is already in "devDependencies". Please remove existing entry first before adding it to "peerDependencies".
```

Reply:
I guess this is an order of operations thing. If I remove jest, then add as a peer first, then a dev, it doesn't show a warning.

Reply:
yeah, that sounds like a bug in the package manager, perfectly legal (and you essentially _always_ want) to have it in both stanzas